modules/atomic-widgets/prop-types/background-image-overlay-prop-type.php

```diff
@@ -3,6 +3,7 @@
 namespace Elementor\Modules\AtomicWidgets\PropTypes;
```

Comment:
I think the frontend transformer is still missing. I am not sure if this part of the task was finished already.
We will work on it in the position task.